pkgs/applications/version-management/gitless/default.nix

```diff
@@ -21,6 +21,7 @@ python3.pkgs.buildPythonApplication rec {
     sh
     pygit2
     clint
```

Comment:
```suggestion
```

I think you can also remove clint dependency as it has been removed upstream from the requirements.txt, see https://github.com/goldstar611/gitless/commit/516d96187cad6f1e4c5dea6861e625c15be41ca5

Reply:
Same for sh: https://github.com/goldstar611/gitless/commit/b4755d742084ab553292a51a19f922665e645c5a#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552

Reply:
Confirmed, still builds and runs with both of those removed, thanks!